auto_nag/bugbug_utils.py

```diff
@@ -94,16 +70,11 @@ def get_bugs(self, date='today', bug_ids=[]):
         finally:
             Bugzilla.BUGZILLA_CHUNK_SIZE = old_CHUNK_SIZE
 
-        # Ignore bugs that we didn't manage to classify recently.
-        recent_bugs = self.get_recent_bugs()
-
-        bug_ids = [bug_id for bug_id in bug_ids if bug_id not in recent_bugs]
-
         bugs = bugzilla._download(bug_ids)
         bugs = list(bugs.values())
 
         # Add bugs that we are classifying now to the cache.
-        self.add_recent_bugs(recent_bugs, bugs)
+        self.add_to_cache(bug['id'] for bug in bugs)
```

Comment:
Can you add a comment here explaining why we call it here and not rely on the call in the parent?